codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -574,10 +575,18 @@ picToObj' pic =
                 ]
                 obj
             retVal obj
-        Text cs style font txt -> do
+        Text cs txt -> do
             obj <- init "text"
             setProps
-                [ ("font", pToJSVal $ fontString style font)
+                [ ("font", pToJSVal $ fontString Plain Serif)
+                , ("text", pToJSVal txt)
+                ]
+                obj
+            retVal obj
+        StyledText cs style font txt -> do
+            obj <- init "styledText"
+            setProps
+                [ ("font", pToJSVal $ fontString Plain Serif)
```

Comment:
I don't think you meant to say that all styledText is Plain and Serif.

Reply:
Not a fan of Plain Serif? :D Made the changes. Thank you!